src/Acuminator/Acuminator.Analyzers/Analyzers/BQL/BqlParameterMismatchAnalyzer.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
```

Comment:
I suggest to remove commented code

Reply:
This is commented diagnostic which I'll use in the future, so it will be in use, just a bit later

Reply:
By the way, unit tests run succesfully